launch/husky_ur5_upload.launch

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
```

Comment:
Instead, chain off description.launch with an arg for ur5_enabled, that gets passed to description xacro